ocs_ci/utility/version.py

```
@@ -74,6 +74,7 @@ def get_semantic_version(
 VERSION_4_20 = get_semantic_version("4.20", True)
 VERSION_4_21 = get_semantic_version("4.21", True)
 VERSION_4_22 = get_semantic_version("4.22", True)
+VERSION_4_23 = get_semantic_version("4.23", True)
```

Ruff (0.16.0)

[warning] Boolean positional value in function call

(FBT003)